docs/guide/writing/style-guides.md

```diff
@@ -49,34 +51,104 @@ If you belong to an open-source community or NGO, you can consider the following
 
-### Thinking about accessibility
+## Style for developers
+
+Developer documentation often have a specific set of rules in order to best meet their needs.
+
+### Code samples
+
+Here are some example guides for code samples
+
+- [Google code samples](https://developers.google.com/style/code-samples)
+- [Ruby / other languages](http://guides.rubyonrails.org/api_documentation_guidelines.html)
+- [Al language snippets](https://bocoup.com/blog/documenting-your-api)
+
+## Thinking about accessibility
 
 It's important to consider accessibility in your style guide to ensure the content you produce can be best understood by all readers. Writing for accessibility includes making sure copy can be read by screenreaders, content organization, style and color of text emphasis, and more.
 
-A few great resources:
+### Accessibility resources
 
-- [MailChimp's style guide](https://styleguide.mailchimp.com/writing-for-accessibility/)
+- [MailChimp's writing style guide](https://styleguide.mailchimp.com/writing-for-accessibility/)
 - [A11Y Style Guide](https://a11y-style-guide.com/style-guide/)
+- [The Accessibility Cheatsheet](https://bitsofco.de/the-accessibility-cheatsheet/) by bitsofcode
 
-### More info is needed for the following
+Relevant talks from Write The Docs:
 
-#### User Guide / Manuals
+- [A11y-Friendly Documentation](https://www.writethedocs.org/videos/prague/2018/a11y-friendly-documentation-carolyn-stransky/) at Write The Docs Prague 2018
+- [Inclusive Tech Docs - TechComm Meets Accessibility](http://www.writethedocs.org/videos/eu/2015/inclusive-tech-docs-techcomm-meets-accessibility-rmatic/) at Write the Docs EU 2015 
 
-#### Tutorials / How-To / FAQ
+## Content guidelines
 
-#### Concepts, Overview
+It's important to create consistency in your content types. This allows you to manage your audience's expectations for what they will learn on any given page.
 
-#### Code Snippets
+### API documentation
 
-- [Google code samples](https://developers.google.com/style/code-samples)
-- [Ruby / other languages](http://guides.rubyonrails.org/api_documentation_guidelines.html)
-- [Al language snippets](https://bocoup.com/blog/documenting-your-api)
+Clear, well-formated, and detailed API documentation is the key for developers to quickly consume and implement your API. It is also key to helping developers understand what happens when an API call is mad, and in the case of failure, understand what went wrong and how to fix it.
+
+The best API documentation is often the result of a designed API. Documentation cannot fix a badly planned API, and it is best to work on developing the API and the documentation concurrently.
+
+[Our guide to API documentation](https://www.writethedocs.org/guide/api/about-api-documentation/).
```

Comment:
Should we just fold this in? There doesn't feel enough in that link to call it a Guide. Or am I overthinking?

Reply:
And by fold in I mean merge.

Reply:
I agree. I'll go ahead and do that. Updates shortly.

Reply:
I've copied the relevant content and no longer link to that page. I've also removed it from the repo.

That said, there are still other files in the API directory that don't necessarily belong in the style guide.